polygon/heimdall/milestone.go

```diff
@@ -25,6 +25,10 @@ func (m *Milestone) RawId() uint64 {
 	return uint64(m.Id)
 }
 
+func (m *Milestone) SetRawId(id uint64) {
+	return
```

Comment:
Is it a good idea to leave these unimplemented since they are unused? Or should I implement them anyway to keep things coherent in case we do use it down the road?

Reply:
let's have panic(unimplemented) for now